### DISCUSSION TITLE
Foreign Subsidy Regulation - only a must for winners?

### DISCUSSION THREAD

**Discussion by EvijaMozga** [OP-TED/eForms-SDK#1050] [Q&A] [answered]
Dear TED team,

According to eForms Guideline on the Foreign Subsidy Regulation: https://code.europa.eu/eproc/eforms/docs/-/blob/main/guides/gde_001_fsr.md It is necessary to provide the Foreign Subsidy Measure BT-682 only for the winning tenderer. To increase transparency, with eForms it is possible to provide this information for all tenderers. This will help buyers and policy makers to understand participation of tenderers who receive foreign subsidies.

Currently, SDK version 12 is available, which includes FSR, and it shows that it should be indicated against the tender. Do we understand correctly that, nevertheless, the FSR only needs to be indicated against the winner (not every tender in case we opt for not implementing this field for every tender)?

Thank you in advance!

Kind regards,

Evija Mozga
Procurement Monitoring Bureau

**Comment by YvesJo** [comment, marked as answer]
Hi,
This information needs to be reported for all LotTenders of the notice (winning and non-winning), unless the LotTender is neither submitted for a Lot for which Foreign Subsidies Regulation (BT-681-Lot) is set to 'true', nor for a Group of Lots containing lots for which Foreign Subsidies Regulation (BT-681-Lot) is set to 'true'. 
Providing information on non-winning tenders is optional and as soon it is done and if the lot is subject to FSR, then BT-682 also has to be specified.

**Comment by EvijaMozga** [comment]
Dear TED Team,

Thank you for the previous reply. We have additional question:
This field must be indicated in the Competition notice, but it is also planned to be implemented in the Result notices. Is it intended that the value of this field can be changed in the Result announcements? For example, if the planned contract price does not exceed 250 millions and the value 'no' is indicated in the field BT-681 of the Competition notice, but the actual contract price exceeds 250 millions — should the value 'yes' be indicated in the Result notice (in the field BT-681), or should this value remain unchanged, and no additional information related to FSR be provided in the Result notice?

Thank you in advance!

Kind regards,

Evija Mozga
Procurement Monitoring Bureau

**Reply by YvesJo** [reply]
Hi,
for this business, non-SDK related question, please get directly in touch with the helpdesk.
KR

**Reply by YvesJo** [reply]
Additionally, be aware that there’s currently no technical restriction/rule that could restrict your options. Additionally to contacting the helpdesk, you may raise your concern to DG GROW at https://code.europa.eu/eproc/eforms/crs/-/issues.
KR
